.github/labeler.yml

```
@@ -1,3 +1,4 @@
-possible stability impact:
-- libfdo-data/fdo_data.h
-- libfdo-data/libfdo-data-go.doc
+"possible stability impact":
+  - changed-files:
+    - any-glob-to-any-file: "libfdo-data/fdo_data.h"
+    - any-glob-to-any-file: "libfdo-data/libfdo-data-go.doc"
```

This should be a YAML array now; [see the new syntax](https://github.com/actions/labeler?tab=readme-ov-file#match-object)

```suggestion
    - any-glob-to-any-file: 
        - "libfdo-data/fdo_data.h"
        - "libfdo-data/libfdo-data-go.doc"
```